src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,13 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery[] list = new Lottery[3];
+        list[0] = new Lottery();
+        list[1] = new Lottery();
+        list[2] = new Lottery();
+        for (Lottery sample : list) {
+            System.out.println(sample.getRandomBall());
+        }
     }
```

Comment:
You can use a for loop to initialize and use the `Lottery` objects in the array. This will make the code more scalable and maintainable.